src/ascent/runtimes/flow_filters/ascent_runtime_blueprint_filters.cpp

```diff
@@ -174,6 +177,81 @@ BlueprintVerify::execute()
     set_output<DataObject>(d_input);
 }
 
+//-----------------------------------------------------------------------------
+BlueprintPartition::BlueprintPartition()
+:Filter()
+{
+// empty
+}
+
+//-----------------------------------------------------------------------------
+BlueprintPartition::~BlueprintPartition()
+{
+// empty
+}
+
+//-----------------------------------------------------------------------------
+void
+BlueprintPartition::declare_interface(Node &i)
+{
+    i["type_name"]   = "blueprint_data_partition";
+    i["port_names"].append() = "in";
+    i["output_port"] = "true";
+}
+
+//-----------------------------------------------------------------------------
+bool
+BlueprintPartition::verify_params(const conduit::Node &params,
+                               conduit::Node &info)
+{
+    info.reset();
+    bool res = true;
+
+    if(! params.has_child("target") ||
+       ! params["target"].dtype().is_int() )
+    {
+        info["errors"].append() = "Missing required int parameter 'target'";
+    }
+
+    return res;
+}
+
+
+//-----------------------------------------------------------------------------
+void
+BlueprintPartition::execute()
+{
+    if(!input(0).check_type<DataObject>())
+    {
+        ASCENT_ERROR("blueprint_data_partition input must be a DataObject");
+    }
+
+    DataObject *d_input = input<DataObject>(0);
+    std::shared_ptr<conduit::Node> n_input = d_input->as_node();
+
+    conduit::Node n_output;
```

Comment:
The data object will own the Node and take care of managing when it needs to be deleted.  So, we can change this to conduit::Node *n_output = new conduit::Node();